tst/EnergyPlus/unit/ChillerExhaustAbsorption.unit.cc

```diff
@@ -669,7 +701,11 @@ TEST_F(EnergyPlusFixture, ExhAbsorption_calcHeater_Fix_Test)
     EXPECT_NEAR(thisChillerHeater.ElectricPower, 400.0, 1e-6);
     EXPECT_NEAR(thisChillerHeater.ExhaustInTemp, 350.0, 1e-6);
     EXPECT_NEAR(thisChillerHeater.ExhaustInFlow, 0.5, 1e-6);
-    EXPECT_NEAR(thisChillerHeater.ExhHeatRecPotentialHeat, 87087.5769469, 1e-6);
+
+    Real64 const CpAir = Psychrometrics::PsyCpAirFnW(exhaustInHumRate);
+    Real64 const epectedExhHeatRecPotentialHeat = exhaustInMassFlowRate * CpAir * (exhaustInTemp - absLeavingTemp);
+    EXPECT_NEAR(87891.51, epectedExhHeatRecPotentialHeat, 0.01);
+    EXPECT_NEAR(epectedExhHeatRecPotentialHeat, thisChillerHeater.ExhHeatRecPotentialHeat, 0.01);
```

Comment:
Check the expectedExhHeatRecPotentialHeat

```suggestion
    Real64 const CpAir = Psychrometrics::PsyCpAirFnW(exhaustInHumRate);
    Real64 const expectedExhHeatRecPotentialHeat = exhaustInMassFlowRate * CpAir * (exhaustInTemp - absLeavingTemp);
    EXPECT_NEAR(87891.51, expectedExhHeatRecPotentialHeat, 0.01);
    EXPECT_NEAR(expectedExhHeatRecPotentialHeat, thisChillerHeater.ExhHeatRecPotentialHeat, 0.01);
```